WalletWasabi.Gui/Controls/WalletExplorer/ClosedWalletViewModel.cs

```diff
@@ -0,0 +1,60 @@
+using AvalonStudio.Extensibility;
+using ReactiveUI;
+using Splat;
+using System.IO;
+using System.Linq;
+using System.Reactive;
+using System.Threading;
+
+namespace WalletWasabi.Gui.Controls.WalletExplorer
+{
+	public class ClosedWalletViewModel : WalletViewModelBase
+	{
+		public ClosedWalletViewModel(string path) : base(Path.GetFileNameWithoutExtension(path))
+		{
+			OpenWalletCommand = ReactiveCommand.CreateFromTask(async () =>
+			{
+				IsBusy = true;
```

Comment:
This flag should be switch to false in a `finally` block.

Reply:
done